x-pack/plugins/fleet/common/experimental_features.ts

```diff
@@ -15,6 +15,7 @@ export const allowedExperimentalValues = Object.freeze({
   createPackagePolicyMultiPageLayout: true,
   packageVerification: true,
   showDevtoolsRequest: true,
+  showRequestDiagnostics: false,
```

Comment:
Added feature flag to hide Request diagnostics action by default at 3 places: Agent list single action, bulk action, Agent details action.
For local testing, set this boolean to true.

Feature flag off:
<img width="1323" alt="image" src="https://user-images.githubusercontent.com/90178898/200532316-fae45671-c79e-4df9-941f-8c50d5ff2254.png">
<img width="1463" alt="image" src="https://user-images.githubusercontent.com/90178898/200532388-4c854657-59f6-4ee6-bbe6-6906eb049e37.png">
<img width="1547" alt="image" src="https://user-images.githubusercontent.com/90178898/200532460-ed847d3b-2d9d-4276-85a5-c86366fa1459.png">

Toggle on:
<img width="1323" alt="image" src="https://user-images.githubusercontent.com/90178898/200532539-fb860e70-25f5-4667-8aa6-5038d01491e3.png">

Reply:
Thanks for adding this!